### DISCUSSION TITLE
OK to update practices via a GH Pull request

### DISCUSSION THREAD

**Discussion by alecthegeek** [openpracticelibrary/openpracticelibrary#2205] [Q&A]
I am struggling to use the CMS editor -- It won't format sub lists correctly.

Can I submit a PR here with updated content?

**Comment by alecthegeek** [comment]
Well I figured it was easier to ask for forgiveness that beg for permission 🤣 

see #2206

**Comment by kattekrab** [comment]
Oh my goodness, sorry @alecthegeek - I did not see this, which means I need to look at my notificatioin settings I think.  Better late than never, and we got your changes through.  yes, it's ok to use a PR, but the autotests can't handle it.  All good in the end, and thank you!!!
